onnxruntime/contrib_ops/rocm/bert/gemm_fast_gelu_impl.cu

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "contrib_ops/rocm/bert/gemm_fast_gelu_impl.h"
+
+#include <hip/hip_fp16.h>
+
+#include "contrib_ops/rocm/bert/gemm_fast_gelu_tunable_op.h"
+#include "core/providers/rocm/tunable/gemm_common.h"
+
+using onnxruntime::rocm::tunable::blas::BlasOp;
+
+namespace onnxruntime {
+namespace contrib {
+namespace rocm {
+
+template <typename T>
+Status LaunchGemmFastGeluKernel(bool tuning,
```

Comment:
Can you document a bit about the memory convention (row-major or column majored), or better put it into a namespace so that it is clear to the caller.

Reply:
Adding comment is better now. I plan to add function` colum_major::gemmfastgelu` / `row_major::gemmfastgelu` when I add ck gemmfastgelu, like `colum_major::Gemm` and `row_major::Gemm`. And the `ComputeInternal` will call `colum_major::gemmfastgelu` directly instead of `LaunchGemmFastGeluKernel`. It will more clear.